src/main/java/org/wso2/carbon/connector/pojo/ConnectionConfiguration.java

```diff
@@ -311,4 +313,26 @@ public Boolean getRequireAuthentication() {
 
         return requireAuthentication;
     }
+
+    public void setEnableOAuth2Authentication(String enableOAuth2Authentication) {
+
+        if (!StringUtils.isEmpty(enableOAuth2Authentication)) {
```

Comment:
Since `Boolean.parseBoolean` is null safe, Do we need this check?

Reply:
Fixed.